pkg/cmdutil/helpers.go

```diff
@@ -74,11 +77,25 @@ func InstallKlog(cmd *cobra.Command) {
 	}
 }
 
-func GetLoglevel() string {
+func getLoglevelOrDefault() (int, error) {
 	f := flag.CommandLine.Lookup("v")
-	if f != nil {
-		return f.Value.String()
+	if f == nil {
+		return 0, errors.New(`can't lookup klog "v" flag`)
 	}
 
-	return ""
+	s := f.Value.String()
+	if len(s) == 0 {
+		return 0, nil
```

Comment:
nit: shouldn't a default have *some* verbosity?

Reply:
the default was always 0, this doesn't change it. we only usually set it to 2 explicitly in the deployments